content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md

```diff
@@ -841,7 +841,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.3/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```

Comment:
This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

Reply:
No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.